### PR TITLE
Phase2-hgx208 Correct V11 geometry for wafer types and many cfg's for correct scenarios

### PR DESCRIPTION
**Pull request by bsunanda**
Correct V11 geometry for wafer types and many cfg's for correct scenarios

#### PR description:

Tried out the phase2 scenarios in the matrix

#### if this PR is a backport please specify the original PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by deguio** [comment]
hi @bsunanda 
after the last commit (`df7da23`) things look good from what I see:
- [V11_D46 on CMSSW_11_0_X_2019-07-26-2300+27555+27622](http://deguio.web.cern.ch/deguio/hgcal/geometry/geom_output_v11_D46_CMSSW_11_0_X_2019-07-26-2300+27555+27622/?match=xy*_9_&depth=1)

to be compared with:
- [V10_D41 on CMSSW_11_0_0_pre4](http://deguio.web.cern.ch/deguio/hgcal/geometry/geom_output_v10_D41/?match=xy*_9_&depth=1)

File: Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc [hunk 1]
```diff
@@ -281,8 +281,10 @@ void DDHGCalEEAlgo::positionSensitive(const DDLogicalPart& glog,
       int iv = std::abs(v);
       int nr = 2 * v;
       int nc = -2 * u + v;
-      double xpos = xyoff.first + nc * r;
-      double ypos = xyoff.second + nr * dy;
+      double xps0 = nc * r;
```
Comment (kpedro88) [inline Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc]: it seems like these new intermediate variables are unnecessary now?

Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc]: No they are not. I was waiting to see Federico's OK

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@fabiocos @smuzaffar  Why the code-checking is stuck?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by fioriNTU** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 Please approve and integrate this PR. i have 2 PR's one to be signed by DB and other by L1. usually they are both slow

**Comment by kpedro88** [comment]
+upgrade

**Comment by kpedro88** [comment]
+1

**Pull request merged**